packages/eventual-send/jsconfig.test.json

```diff
@@ -0,0 +1,4 @@
+{
+  "extends": "./jsconfig.json",
+  "include": ["test/test-types.js"]
```

Comment:
I don't remember, do this extend or replace the list of included files?

Reply:
Good catch! It replaces according to https://www.typescriptlang.org/tsconfig#extends

So this change made it so _only_ test-types is checked. Regret I missed that.